packages/headless/src/controllers/insight/facets/facet/headless-insight-facet.ts

```diff
@@ -0,0 +1,174 @@
+import {
+  executeSearch,
+  fetchFacetValues,
+} from '../../../../features/insight-search/insight-search-actions';
+import {
+  logFacetClearAll,
+  logFacetUpdateSort,
+  logFacetShowMore,
+  logFacetShowLess,
+} from '../../../../features/facets/facet-set/facet-set-analytics-actions';
```

Comment:
analytics actions need to be insight analytics actions
see https://github.com/coveo/ui-kit/pull/2147